Mage.Sets/src/mage/cards/s/SoulSeparator.java

```diff
@@ -59,7 +59,7 @@ public SoulSeparator copy() {
 
     public SoulSeparatorEffect() {
         super(Outcome.PutCreatureInPlay);
-        this.staticText = "Create a black Zombie creature token with power equal to that card's power and toughness equal that card's toughness";
+        this.staticText = "Put a black Zombie creature token with power equal to that card's power and toughness equal that card's toughness";
```

Comment:
This was actually correct wording. We are creating tokens instead of putting them onto battlefield since Kaladesh (I think)

Reply:
I checked the rules and you're correct. I will fix it and add a note in the code. Thanks for pointing this out!